packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1059,8 +1061,19 @@ export class ContainerRuntime
 			(schema) => {
 				runtime.onSchemaChange(schema);
 			},
+			{ minVersionForCollab },
+			logger,
 		);
 
+		// If the minVersionForCollab for this client is greater than the existing one, we should use that one going forward.
+		const existingMinVersionForCollab =
```

Comment:
Isn't this redundant with DocumentSchema logic?  Can you just read the value off that thing?

Reply:
I think it's a bit tricky since the purpose of `existingMinVersionForCollab`/`updatedMinVersionForCollab` is to to log in the `ContainerLoadStats` event. I believe we send that event before we process any schema change messages. Maybe it makes more sense to log whatever the session schema is at the time of the event?